src/protocols/aeternity/views/TransactionDetails.vue

```diff
@@ -319,10 +320,11 @@ export default defineComponent({
     onMounted(async () => {
       let rawTransaction = getTransactionByHash(activeAccount.value.address, hash);
 
-      if (!rawTransaction || rawTransaction.incomplete) {
+      // Claim transactions have missing data that needs to be fetched from the middleware
+      if (!rawTransaction || rawTransaction.incomplete || rawTransaction.claim) {
```

Comment:
Do we need to fetch `claim` transaction from the middleware, if we already fetched it from backend?

Reply:
Yes, the backend returns faulty and incomplete data

Reply:
I see, incomplete transaction:
 - doesn't have `Gas price`
 - doesn't have `Gas used`
 - have a wrong timestamp